template/blast.config.js

```diff
@@ -1,16 +1,15 @@
-/* eslint-disable quote-props */
-const accounts = require('./accounts.json')
-const privateAccounts = require('./private-accounts.json')
+/* eslint-disable object-curly-newline */
 module.exports.config = {
-  name: 'template',
-  network: 'cudos',
-  defaultAccount: 'account1',
-  accounts: accounts,
-  privateAccounts: privateAccounts,
   addressPrefix: 'cudos',
+  gasPrice: '250acudos',
+  localNetwork: 'http://localhost:26657',
```

Comment:
can we move 'localNetwork' in networks {}?

Reply:
this localNetwork is designed to be read-only (furthermore, changing it will break the tool) so I will move it into a new internal blast config